test/migrate-cli.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
```

Comment:
wouldn't it be simpler to just write a Jake (https://www.npmjs.com/package/jake) script within usual unit test?